packages/nimble-components/src/icon-base/icon-metadata.ts

```diff
@@ -223,15 +226,15 @@ export const iconMetadata: {
     IconFile: {
         tags: ['file-tdms']
     },
+    IconFileArrowCurvedRight: {
+        tags: ['export']
```

Comment:
I guess this was mentioned at stand-up but looks like file-arrow-curved-right was previously file-export and had tag download. Just double checking if this should be download or export

Reply:
Download was the wrong metaphor, but you reminded me that the primary metaphor the visual designer wanted for this icon was `file-extract`.